docs/deploy.md

```diff
@@ -82,6 +82,117 @@ A simple configuration using `global_deploy.conf` could look like:
       }
     }
 
+Running the deploy task with this config would result in a single instance of `servicename`
+being deployed to the host `ari-staging.allenai.org`, with code placed in the remote directory
+`/local/deploy/servicename`.
+
+A more complex configuration would involve deploying multiple instances of a host in parallel. An
+example configuration achieving this would be:
+
+    staging = {
```

Comment:
FWIW you can also enclose a block in triple-` to get code formatting. It's usually slightly easier than messing with indent.